### PR TITLE
Document VirtualBox discard option

### PR DESCRIPTION
**Pull request by agners**
## Proposed change

Document VirtualBox discard option which frees up unused space automatically.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — APPROVED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
Rendering breaks:

CleanShot 2022-05-02 at 12 45 39

Please ensure you have a new line between HTML & Markdown contexts (so tha parser is able to switch).

**Comment by agners** [comment]
Hm, I guess in theory we could use some manual HTML inside the info block like:

```html
<div class="note info">
  By default VirtualBox does not free up unused disk space. To automatically shrink the vdi disk image
  the `discard` option must be enabled:
  <pre class="highlight language-bash">
    <code data-lang="bash" class=" language-bash">VBoxManage storageattach --storagectl "SATA" --port 0 --device 0 --nonrotational on --discard on</code>
  </pre>
</div>
```

Render somwhat awkward though:
image

Maybe just don't use the info block? Thoughts?

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -207,11 +207,18 @@ _All these can be extended if your usage calls for more resources._
...
     4. Edit the “Settings” of the VM and go “System” then “Motherboard” and select “Enable EFI”
     5. Then go to “Network” “Adapter 1” choose “Bridged Adapter” and choose your Network adapter
     6. Then go to “Audio” and choose “Intel HD Audio” as Audio Controller.
+    <div class="note info">
+    By default VirtualBox does not free up unused disk space. To automatically shrink the vdi disk image
+    the `discard` option must be enabled:
+    ```bash
+    VBoxManage storageattach <VM name> --storagectl "SATA" --port 0 --device 0 --nonrotational on --discard on
+    ```
+    </div>
```
Comment (ludeeus) [inline source/_includes/installation/operating_system.md]:
```suggestion
    <div class="note info">

    By default VirtualBox does not free up unused disk space. To automatically shrink the vdi disk image
    the `discard` option must be enabled:
    ```bash
    VBoxManage storageattach <VM name> --storagectl "SATA" --port 0 --device 0 --nonrotational on --discard on
    ```

    </div>
```

Just needs some space to render properly

Comment (frenck) [reply source/_includes/installation/operating_system.md]: Yup, as I wrote:

> Please ensure you have a new line between HTML & Markdown contexts

:)

Comment (ludeeus) [reply source/_includes/installation/operating_system.md]: That you did 😄  (I missed that part)

**Review by frenck — APPROVED** [review]
Thanks, @agners 👍

**Pull request merged**